crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -491,6 +493,8 @@ impl TimelineAction {
                                         new_content,
                                     )),
                                     edit_json,
+                                    // TODO maybe we could provide it?
```

Comment:
Has it been addressed?

Reply:
It can be done, I'll open a followup PR to keep changes tidy.